### PR TITLE
[author] [new] added cookieconsent2@1.0.7 with git auto-update

### PR DESCRIPTION
**Pull request by Dolondro**
Hi!
Watch: 20 / Star: 158 / Fork: 58
Git Repository: https://github.com/silktide/cookieconsent2
The initial content is pulled from the build folder and (if I've added it correctly) it should be enabled to do git auto-update.

Kind regards!
Doug

### CODE REVIEW

File: ajax/libs/cookieconsent2/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "name": "cookieconsent2",
+  "version": "1.0.7",
+  "description": "Silktide's cookie consent tool.",
+  "main": "cookieconsent.js",
+  "homepage": "http://silktide.com/tools/cookieconsent2",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/silktide/cookieconsent2.git"
+  },
+  "scripts": {
```
Comment (PeterDaveHello) [inline ajax/libs/cookieconsent2/package.json]: Please remove `scripts` here, thanks!

File: ajax/libs/cookieconsent2/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "cookieconsent2",
+  "version": "1.0.7",
+  "description": "Silktide's cookie consent tool.",
+  "main": "cookieconsent.js",
```
Comment (PeterDaveHello) [inline ajax/libs/cookieconsent2/package.json]: Please remove `main` here, thanks!

**Comment by PeterDaveHello** [comment]
Please add `"filename": "cookieconsent.min.js"` in package.json

**Comment by Dolondro** [comment]
Thanks for the quick response!

I've made the changes you've requested

Kind Regards,
Doug

**Comment by PeterDaveHello** [comment]
Would you please squash them into only one commit as our document says? Thanks.

**Comment by Dolondro** [comment]
That took a surprising amount of effort, but done!

Regards,
Doug

**Comment by PeterDaveHello** [comment]
Merge now, two notes:

1. If there may be other directory in `build` folder in future, should change `*` to `**/*` for recursively grabbing the files.
2. You can add `"keywords"` for it if you want!

**Comment by Dolondro** [comment]
Awesome! Thank you very much! =D

**Comment by PeterDaveHello** [comment]
Np

**Pull request merged**